src/server/scripts/Kalimdor/zone_silithus.cpp

```diff
@@ -1148,7 +1149,9 @@ class go_wind_stone : public GameObjectScript
             go_wind_stoneAI(GameObject* go) : GameObjectAI(go) { }
 
             private:
-                uint8 GetPlayerRank(Player* player) // For random summoning
+                bool isSummoning;
```

Comment:
Please initialized class member fields in the constructor to avoid uninitialized values (even if `Reset()` is assigning a value

Reply:
Yeah, I thought the way of declaring them uninitialized was a bit weird too, but I took example from the rest of the script and thought that was "The Way To Do It™". I'll change rest of them too.

Reply:
I initialized all values found in the script's classes except for the ObjectGuid ones. By a quick glance it mostly seems that the ObjectGuids are checked to exist before trying to access them, but I didn't have a meticulous look through the whole script.

Reply:
ObjectGuid has a default constructor that ensures it is always set to 0 so it does not need any additional initialization code

Reply:
Okay, thanks for the review :)